### PR TITLE
Chat functionality

### PR DESCRIPTION
**Pull request by castaneda-ed**
Inbox component now with socket io and some css for it.

### CODE REVIEW

**Review by beefchief-code — APPROVED** [review]
looks awesome!

File: src/styles/Inbox.css [hunk 1]
```diff
@@ -12,15 +12,21 @@
 .recieved {
   background-color: var(--convene-gray);
   color: var(--convene-blue);
+  word-wrap: break-word;
+  overflow-wrap: break-word;
 }
```
Comment (beefchief-code) [inline src/styles/Inbox.css:18]: could move the word-wrap and overflow-wrap from "sent" and "received" to "message" but no big deal

Comment (castaneda-ed) [reply src/styles/Inbox.css:18]: You right! I was just doing changes so fast didn't  realize that

**Review by wangbertha — APPROVED** [review]
This is awesome!

**Pull request merged**